data-sources/jmeter/config.yml

```diff
@@ -0,0 +1,31 @@
+id: jmeter
+displayName: JMeter
+description: |
+  Apache JMeter is open source software, a Java application designed to load test functional behavior and measure performance. It was originally designed for testing Web Applications but has since expanded to other test functions.
```

Comment:
It would be for both issues (240 characters and markdown usage). Other data sources with descriptions exceeding 240 characters were likely created before we added that extra validation. 

It seems also the data source is using an invalid category term `open source testing` the one you maybe looking for is `open source monitoring`

[Valid category terms can be found here](https://api.newrelic.com/graphiql?#query=%7B%0A%20%20actor%20%7B%0A%20%20%20%20nr1Catalog%20%7B%0A%20%20%20%20%20%20categories%20%7B%0A%20%20%20%20%20%20%20%20terms%0A%20%20%20%20%20%20%20%20displayName%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A)

Reply:
Thanks for the pointers. Made both the changes and committed it.